partners.yml

```diff
@@ -457,6 +457,10 @@ components:
           type: boolean
           description: Indicate if customer accepted can be profiled by Moank to take a credit decision
           example: true
+        bankAccountId:
+          type: string
+          description: Tink code fetched with the Tink link.
```

Comment:
Here it should not be the tink code fetched previously but it should be the id of the account chosen by the customer from the list of the account we are returning when they call the `send-code` endpoint ;)